server/Routes/dogs.js

```diff
@@ -95,4 +96,25 @@ router.get(
   },
 );
 
+//add dog to contest
+router.post('/register-dog/:contestId', async (req, res) => {
+  try {
+    const contest = await Contest.findById(req.params.contestId);
+    contest.obedienceClasses
+      .find(
+        (obedienceClass) =>
+          obedienceClass.classNumber == req.body.obedienceClass,
```

Comment:
Robicie jakas walidacje tego co wskakuje na serwer?

Reply:
Ultimately body of this post will go to results.post a there is an authorization middleware there.

Reply:
Authorization is one issue, I mean validation of body for example. Imagine that obedience class in your db is supposed to be an array, but instead an object or something else is pushed. Is there a place where the body that you send in the request is checked and validated against what its supposed to be?